sdk/go/Makefile

```diff
@@ -1,7 +1,7 @@
 PROJECT_NAME     := Pulumi Go SDK
 LANGHOST_PKG     := github.com/pulumi/pulumi/sdk/v2/go/pulumi-language-go
 VERSION          := $(shell ../../scripts/get-version HEAD)
-PROJECT_PKGS     := $(shell go list ./pulumi/... ./pulumi-language-go/... ./common/...| grep -v /vendor/ | grep -v templates)
+PROJECT_PKGS     := $(shell go list ./pulumi/... ./pulumi-language-go/... ./common/... ./x/...| grep -v /vendor/ | grep -v templates)
```

Comment:
The current plan is to remove the `/x/` subpackage before the "1.0" moment. Then it will be versioned in step with core pulumi pulumi as `github.com/pulumi/sdk/v2/go/auto`